src/defaultConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { ContextConfig } from "./types/contextConfig";
+import { FeaturesRegistry } from "./impl/featuresRegistry";
+import { EthTxBuilder } from "./impl/txBuilders/ethTxBuilder";
+import { GithubDappletProvider } from './impl/providers/githubDappletProvider';
+
+const DEFAULT_CONFIG: ContextConfig = {
+    provider: new GithubDappletProvider(),
+    features: new FeaturesRegistry([EthTxBuilder])
```

Comment:
There is a name clash between the term `feature` here and injected UXs. 
I would suggest, we can leave the term `feature` here if we rename it there.

Reply:
I would name it as a module or package... 
It seems for me, a feature is a strange word in this context.